README.md

```diff
@@ -1,2 +1,2 @@
-# oneAPI-samples
+oneAPI-samples
```

Comment:
Looks like you are changing header to a paragraph, is this intentional?

Reply:
This is just a place holder, we are working on a readme, so do not approve the merge as of yet